SoftLayer/managers/ticket.py

```diff
@@ -22,24 +22,30 @@ def __init__(self, client):
         self.account = self.client['Account']
         self.ticket = self.client['Ticket']
 
-    def list_tickets(self, open_status=True, closed_status=True):
+    def list_tickets(self, open_status=True, closed_status=True, limit=None, all_tickets=None):
         """List all tickets.
 
         :param boolean open_status: include open tickets
         :param boolean closed_status: include closed tickets
+        :param int limit: result limit
         """
         mask = """mask[id, serviceProviderResourceId, title, assignedUser[firstName, lastName], priority,
                   createDate, lastEditDate, accountId, status, updateCount]"""
 
         call = 'getTickets'
+        # print('All Checks ',open_status, closed_status,all_tickets)
         if not all([open_status, closed_status]):
             if open_status:
                 call = 'getOpenTickets'
             elif closed_status:
                 call = 'getClosedTickets'
             else:
                 raise ValueError("open_status and closed_status cannot both be False")
-        return self.client.call('Account', call, mask=mask, iter=False, limit=100)
+
+        if all([open_status, all_tickets]):
+            return self.client.call('Account', call, mask=mask, iter=all_tickets, limit=limit)
+
+        return self.client.call('Account', call, mask=mask, iter=False, limit=limit)
```

Comment:
You don't need this if statement if we set the default for `all_tickets` to False.  We also want to allow users the options to get all closed tickets as well, and this change will allow that.

```suggestion
        return self.client.call('Account', call, mask=mask, iter=all_tickets, limit=limit)
```